setup.cfg

```diff
@@ -29,8 +29,8 @@ project_urls =
 [options]
 packages = find:
 install_requires =
-    synapseclient>=2.7.0, <3.0.0
-    pandas>=1.0,<1.5.0
+    synapseclient>=3.0.0, <4.0.0
```

Comment:
Same comment here about synapseclient, also either in this ticket or another ticket, bump the python versions we support.